pkg/apis/crd/v1alpha1/types.go

```diff
@@ -289,6 +292,17 @@ type Advertisements struct {
 	Egress *EgressAdvertisement `json:"egress,omitempty"`
 }
 
+type Confederation struct {
+	// Enabled indicates whether BGP confederation is enabled.
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
Do we need an "enabled" field or should the `confederation` field be a pointer / optional in `BGPPolicySpec`?

Reply:
We can remove `Enabled`, and how about a pointer `confederation`?